exp_build/plugin.js

```diff
@@ -70,15 +118,8 @@ class Plugin {
     );
 
     let plugins = [];
-    if (!fs.existsSync("plugins")) {
-      shell.exec("mkdir plugins");
-    }
-
     expScopePlugins.forEach((plugin) => {
       try {
-        shell.cd("plugins");
-        prepareRepo(plugin);
```

Comment:
Why is the call to prepareRepo removed?

Reply:
A new method was created in the Plugin class called loadAllPlugins which is called inside the build method of experiment class. Inside loadAllplugins, prepareRepo is called.